docs/releases/202407.mdx

```diff
@@ -15,6 +32,7 @@ displayed_sidebar: releaseSideBar
 **Improvements**
 - **Agents**: Agents will now raise a specific error if they are unable to start a flow run, previously this error looked similar to other retries
 - **Flows**: The [send_email method](../api/GanymedeClass#method-send_email) is now testable in notebooks. No email will be sent while testing, an HTML rendering of the email will be displayed
+- **Dashboards**: Agents, connections, and flow errors are now available as data sources to be used in dashboards.
```

Comment:
what are the table names for agent and connections?

Reply:
I have that in another PR